blocks/key-facts/key-facts.js

```diff
@@ -132,6 +132,13 @@ export default async function decorate(block) {
         col.replaceWith(video);
         videoLinks[0].remove();
 
+        // setPlaybackControls();
+        const playbackControls = document.querySelectorAll('video > button');
```

Comment:
The file has createVideo which calls setPlaybackControls, which does similar thing. Any reason for doing this ?

Reply:
It is now uncommented since it was commented by mistake.